frontend/src/models/Robot.ts

```diff
@@ -14,6 +14,7 @@ export interface Robot {
     name?: string
     model: RobotModel
     serialNumber?: string
+    currentAsset?: string
```

Comment:
```suggestion
    currentAsset: string
```
If its not nullable in the backend it shouldn't be nullable here.

Reply:
done